app/src/main/res/values/dimens.xml

```diff
@@ -219,8 +219,6 @@
     <dimen name="credit_cards_add_credit_card_button_height">48dp</dimen>
     <!-- The horizontal margin for the "Add credit card" text.
          This should be the same value as radio_button_padding_horizontal. -->
```

Comment:
Sorry, missed calling out `credit_cards_add_credit_card_button_height` is not needed as well I think. We can also remove their associated comments.